hack/update-catalog-template.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CATALOG_TEMPLATE=${1:-config/catalog/basic-catalog-template.yaml}
+BUNDLE_IMGS=${2:-}
+YQ=${3:-yq}
+
+if [[ -z "$BUNDLE_IMGS" ]]; then
+    echo "Error: BUNDLE_IMGS is required. Please provide at least one image in the format 'image:tag' or 'image:tag,image:tag'" >&2
+    exit 1
+fi
+
+if ! [[ "$BUNDLE_IMGS" =~ ^[^:]+:[^:,]+([,][^:]+:[^:,]+)*$ ]]; then
+    echo "Error: Invalid BUNDLE_IMGS format. Expected format: 'image:tag' or 'image:tag,image:tag,...'" >&2
+    echo "Got: $BUNDLE_IMGS" >&2
+    exit 1
+fi
+
+YQ_VERSION=$($YQ --version | grep -o 'version.*' | cut -d' ' -f2)
+if [[ ! "$YQ_VERSION" =~ ^v?4\. ]]; then
+    echo "Error: YQ version 4.x.x is required. Found version: $YQ_VERSION" >&2
+    exit 1
+fi
+
+package_name="activemq-artemis-operator"
+
+function add_bundle() {
+    local package_name=$1 img=$2 prev_version=$3
+    local version bundle_name
+    
+    version=$(echo "$img" | cut -d':' -f2)
+    bundle_name="${package_name}.v${version}"
+
+    $YQ -i e "
+        select(.schema == \"olm.template.basic\").entries += [{
+            \"schema\": \"olm.bundle\",
+            \"image\": \"$img\"
+        }]
+    " "$CATALOG_TEMPLATE"
+
+    if [[ -n "$prev_version" ]]; then
+        $YQ -i e "
+            select(.schema == \"olm.template.basic\").entries[] |=
+                select(.schema == \"olm.channel\" and .name == \"upstream\").entries += [{
+                    \"name\": \"$bundle_name\",
+                    \"replaces\": \"$package_name.$prev_version\"
```

Comment:
The package version has the prefix ``
```suggestion
                    \"replaces\": \"$package_name.v$prev_version\"
```